android-app/app/src/main/java/de/dhbw/ka/se/fibo/adapters/CashflowAdapter.java

```diff
@@ -0,0 +1,66 @@
+package de.dhbw.ka.se.fibo.adapters;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.graphics.Typeface;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.math.BigDecimal;
+import java.text.Format;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Locale;
+
+import de.dhbw.ka.se.fibo.Helpers;
+import de.dhbw.ka.se.fibo.R;
+import de.dhbw.ka.se.fibo.models.Cashflow;
+import de.dhbw.ka.se.fibo.models.CashflowType;
+
+public class CashflowAdapter extends ArrayAdapter<Cashflow> {
+
+    private final LayoutInflater inflater;
+
+    public CashflowAdapter(@NonNull Context context, int resource, ArrayList<Cashflow> arrayList) {
+        super(context, resource, arrayList);
+        this.inflater = LayoutInflater.from(context);
+    }
+
+    @NonNull
+    @Override
+    public View getView(int position, @Nullable View convertView, @NonNull ViewGroup parent) {
+        Cashflow cashflow = this.getItem(position);
+        CashflowType cashflowType = cashflow.getType();
+        BigDecimal overallValue = cashflow.getOverallValue();
+
+        @SuppressLint("ViewHolder") // TODO: Use a view holder to improve performance
+        View view = this.inflater.inflate(R.layout.row_layout, null);
```

Comment:
Also, maybe we have the time to resolve this todo. Yet, if it seems out of reach for now, it'd be okay for me to merge. (The technical debt we introduce is negligible for now and we can easily tackle this in the future as well.)

Reply:
We might need to use a ViewHolder when we use a RecyclerView. So if I got it correctly the TODO might get resolved anyways